scib/resources/convert_genes.py

```diff
@@ -0,0 +1,112 @@
+import requests
+from tqdm import tqdm
+
+
+def get_gene_name_from_ensembl(gene_ids: list, species: str):
```

Comment:
Isn't there a simpler way to do it? E.g. using `bionty` Gene [lookup](https://docs.lamin.ai/bionty.gene#bionty.Gene).  It probably does the same under the hood, but if something breaks in API, it is their responsibility to fix it, and not your

Reply:
It can be fine to leave it like that of course, if you don't want to add an additional dependency

Reply:
Good point! With bionty it would probably be more elegant, but I didn't want to include any additional dependencies here and this script is only for reproducibility purposes (i.e. will never be run in the package)